src/EditCell.tsx

```diff
@@ -77,19 +85,16 @@ export default function EditCell<R, SR>({
     };
   }, [cancelFrameRequest, commitOnOutsideClick, commitOnOutsideMouseDown, skipCellFocusRef]);
 
-  function onKeyDown(event: React.KeyboardEvent<HTMLDivElement>) {
+  function handleKeyDown(event: React.KeyboardEvent<HTMLDivElement>) {
+    onKeyDown?.({ mode: 'EDIT', row, column, rowIdx, onClose }, event);
+    if (event.isDefaultPrevented()) return;
     if (event.key === 'Escape') {
-      event.stopPropagation();
```

Comment:
No, it was added because we do not want to invoke parent's `onKeyDown` handler. Now we return if the cell is in edit mode
https://github.com/adazzle/react-data-grid/pull/2833/files#diff-0f9c631e39f2adf9fe111a2bb9ca7fab29327c1d43978939e30d3d36f5177ac4R595